welcome.yaml

```diff
@@ -0,0 +1,132 @@
+- type: text
+  text: https://github.com/HelixLauncher/Art/blob/main/branding/banner-launcher/banner-launcher_96h.png?raw=true
+
+- type: embed
+
+  embeds:
+    - title: Welcome to HelixLauncher
+
+      description: >
+        This is the official Discord server for the HelixLauncher project.
+        
+        
+        HelixLauncher is an open-source Minecraft Launcher written from the ground-up in Rust.
+        We have split HelixLauncher into two main parts, the backend library, which anyone can use for their own launchers
+        and the GUI/CLI packages, which use the library to create a user-friendly Minecraft launcher.
+        
+        
+        We hope you enjoy your time with us!
+
+- type: text
+  text: https://github.com/HelixLauncher/Art/blob/main/branding/banner-guidelines/banner-guidelines_96h.png?raw=true
+
+- type: rules
+
+  rules:
+    Follow Helix's Community Guidelines: >
+      Everyone is expected to read and abide by our [Code of Conduct](https://github.com/HelixLauncher/Community-Guidelines/blob/main/code-of-conduct.md)
+
+    Follow Discord's rules: >
+      As a Discord-based community, we require everyone to follow Discord's
+      [Community Guidelines](https://discord.com/guidelines) and [Terms of Service](https://discord.com/terms)
+
+    Try to speak English: >
+      The common language in our community is English, but we understand that not everyone speaks English at the same
+      level. Try to speak English to the best of your ability, and don't worry if you need to use translation tools. We
+      will not tolerate making fun of people for poor English.
+
+    Listen to & respect all community team members: >
+      Please take requests from community team members seriously and listen when a team member asks you to do
+      something. If you have a problem with what a team member is asking you to do, we advise that you step away from
+      the situation and send a private message to another team member.
+
+    Keep all projects legal, legitimate & appropriate: >
+      We will not tolerate any projects that may break laws, breach a Terms of Service or an EULA, involve sexual
```

Comment:
"breach a Term of Service" or "breach our Terms of Service"

Reply:
now that you mention it, the same issue with basing stuff on "may break laws" that was in the guidelines also applies here